backend/app/scraping/croatia_scraper.py

```
@@ -4,47 +4,181 @@
+            "category": "Tourism"
+        }
+    }
+
+    @staticmethod
+    def categorize_event(title: str, description: str, location: str = "") -> str:
+        """Categorize event based on content analysis using Croatian keywords."""
+        content = f"{title} {description} {location}".lower()
+        
+        # Score each category based on keyword matches
+        category_scores = {}
+        for cat_key, cat_data in CroatiaEventDataTransformer.EVENT_CATEGORIES.items():
```

Ruff (0.12.2)

Loop control variable `cat_key` not used within loop body

Rename unused `cat_key` to `_cat_key`

(B007)